examples/Peripherals/HardwareTimer/countermode_up_down_centered/countermode_up_down_centered.ino

```diff
@@ -0,0 +1,123 @@
+/*
+  Countermode up down centered
+  This example shows how to configure HardwareTimer's counter mode.
+  
+  The following countermodes are demonstrated:
+  LL_TIM_COUNTERMODE_UP
+   _ — ‾ | _ — ‾ | _ — ‾ | _ — ‾ |
+  
+  LL_TIM_COUNTERMODE_DOWN
+   ‾ — _ | ‾ — _ | ‾ — _ | ‾ — _ |
+  
+  LL_TIM_COUNTERMODE_CENTER_UP_DOWN (with over/undeflow callback)
+  _ — ‾ | ‾ — _ | _ — ‾ | ‾ — _ |
```

Comment:
Not sure how this is more accurate.  My code is actually Serial.print'ing those character.  I'm using 3 timers change interrupts.  First one prints `_` at 25%, second prints `—` at 50% and third prints `‾` at 75%.  The `|` is printed in an overflow interrupt. 
Perhaps you are saying what I've done is confusing.  Possibly.

Reply:
More accurate in the sense that it reflects the hardware counter value, with the asymmetry I highlight.
Your figure let suppose it is symmetrical, it is not.
By the way it points me to an issue, your figure tend to show that period of center aligned is twice the one of upcounting. It is wrong:
When timer is configured in upcounting period is ARR+1, so when using Arduino API  `timer->setOverflow(1, HERTZ_FORMAT);`  we take this into account and we calculate ARR so that ARR+1 = requested period
(or as close as possible). Fine.
But when changing counter to center aligned period is 2 * ARR: from 0 to ARR-1 for up direction, from ARR to 1 in down direction (and it is not twice period of up counting like one can intuitively imagine).
So when changing to center aligned, ARR should be recomputed with a different formula.
For High ARR value this may be bnegligeable, but for small value it can be a real issue.